readmeio/api-explorer
# fix: parse example.response.value.$ref rather than render as plain text

## 🧰 What's being changed?
 | 🎫 [Ticket](https://app.asana.com/0/1178132843162889/1148431658605906)|
:---:

If a customer has a `$ref` in their example responses, we currently render these as plain text. This change would find the schema definition for the `$ref` and show the schema instead! 

We're currently only checking if `example.response` exists — if it does, we're returning the `value`. Since the `$ref` can be contained in the `value` field, I've added logic to check for the existence of `$ref` first. If it exists, we're going to call `findSchemaDefinition` on the `$ref` and the oas file.

## 🧪 Testing
1. Find a file that's using a `$ref` in their response examples (or use this one [example with ref.json.zip](https://github.com/readmeio/api-explorer/files/5290347/example.with.ref.json.zip))
2. Pull it into preview.readme.io or bin.readme.com
3. Notice that the endpoint will display the `$ref` as plain text
4. Check on this branch with the same file.
5. It should find the `$ref` and render it correctly.
